guides/getting-started.md

```diff
@@ -38,6 +38,77 @@ The `ng add` command will additionally perform the following configurations:
 
 You're done! Angular Material is now configured to be used in your application.
 
+### Install Angular Material manually
+
+Follow these instructions in case you might need to set-up [Angular Material](https://material.angular.io) on an online code-editor like [StackBlitz](https://stackblitz.com/fork/angular), [CodeSandbox](https://codesandbox.io/s/angular), or [NG-Run](https://ng-run.com/).
```

Comment:
We have a Stackblitz template with Material set up already. https://stackblitz.com/fork/components-issue